### PR TITLE
CCS-15-CreateClothingHandler

### PR DESCRIPTION
**Pull request by RRDarius**
- Creates Route Handler for CreateClothing and updates route config file to reflect this change
- https://cornercasesolutons.atlassian.net/jira/software/projects/CCS/boards/1?selectedIssue=CCS-15

### CODE REVIEW

**Review by jmusone — APPROVED** [review]

File: src/main/java/com/ccs/inventorymanagement/route/CreateClothingHandler.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                        .status(request.getStatus())
+                        .description(request.getDescription())
+                        .brand(request.getBrand())
+                        .color(request.getColor())
+                        .type(request.getType())
+                        .gender(request.getGender())
+                        .size(request.getSize())
+                        .build()))
+                .flatMap(clothing -> ServerResponse.ok()
+                        .body(BodyInserters.fromValue(Response.from(clothing))));
```
Comment (hitchan) [inline src/main/java/com/ccs/inventorymanagement/route/CreateClothingHandler.java]: What happens if you get an empty or an error? What Response do you want. It seems to be missing here

File: src/main/java/com/ccs/inventorymanagement/route/CreateClothingHandler.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                    .condition(clothing.getCondition())
+                    .status(clothing.getStatus())
+                    .description(clothing.getDescription())
+                    .brand(clothing.getBrand())
+                    .color(clothing.getColor())
+                    .apparelType(clothing.getType())
+                    .gender(clothing.getGender())
+                    .size(clothing.getSize())
+                    .build();
+        }
```
Comment (hitchan) [inline src/main/java/com/ccs/inventorymanagement/route/CreateClothingHandler.java]: Your response is missing some data, but if that is intentional, disregard.

File: src/main/java/com/ccs/inventorymanagement/route/CreateClothingHandler.java (hunk 1 above)
Comment (jmusone) [reply src/main/java/com/ccs/inventorymanagement/route/CreateClothingHandler.java]: Fixed, please review.

File: src/main/java/com/ccs/inventorymanagement/route/CreateClothingHandler.java (hunk 2 above)
Comment (jmusone) [reply src/main/java/com/ccs/inventorymanagement/route/CreateClothingHandler.java]: Fixed, please review.

**Review by hitchan — APPROVED** [review]
LGTM

**Pull request merged**